solarforecastarbiter/reports/figures/plotly_figures.py

```diff
@@ -835,23 +835,25 @@ def bar(df, metric):
     data = df[(df['category'] == 'total') & (df['metric'] == metric)]
     y_range = None
     x_axis_kwargs = {}
-    x_values = data['abbrev']
+    x_values = []
+    for val, ser in data[['abbrev']].groupby('abbrev'):
```

Comment:
```suggestion
    # to avoid stacking, add null characters to fx with 
    # same abbreviated name. GH463
    for val, ser in data[['abbrev']].groupby('abbrev'):
```